README.md

```diff
@@ -1,2 +1,28 @@
 # python-package-template
 Cookiecutter template for Python packages
+
+Setup
+-----
+
+Install cookiecutter on your machine
+
+`pip install cookiecutter`
+
+
+Clone this repository
+
+`git clone git@github.com:Thermondo/python-package-template.git`
+
+
+Locally change variables in cookiecutter.json to suit your project
```

Comment:
I think this is not needed. Variables in `cookiecutter.json` are defaults, user can input sth else when running cookiecutter.